src/app/components/shared/ConfirmScaffoldForm/utils/createValuesObject.tsx

```diff
@@ -5,23 +5,28 @@ import { Field } from "../../Form/ScaffoldField"
 const mapField = <T extends RequestBody>(field: Field, key: string, data: T) => {
   const { type, props } = field
   const { label = key, name } = props
-
+ 
   if (name === undefined) return
 
   const v = data[name]
+  if (v === undefined && !name.includes("type_result")) return
 
-  if (v === undefined) return
-
-  const value =
-    type === "ArrayField" ?
-      <ArrayFieldList fields={ v as Array<Record<string, string>> } /> :
-    type === "ComplexSelectField" || type === "ComplexRadioFields" ?
-      (v as Record<string, string>)[(props as { optionLabelField: string }).optionLabelField] :
-    props.hasOwnProperty("options") ?
-      (props as { options: Record<string, unknown> }).options[v as string] :
-      v
+  const value = (type: string, props: any) => {
+    if(type === "ArrayField") {      
+      return <ArrayFieldList fields={ v as Array<Record<string, string>> } />
+    }else if(type === "ComplexSelectField" || type === "ComplexRadioFields"){
+      return(v as Record<string, string>)[(props as { optionLabelField: string }).optionLabelField]
+    } else if (props.hasOwnProperty("options")) {
+      return (props as { options: Record<string, unknown> }).options[v as string] 
+    } else if( name.includes("type_result") ) {
```

Comment:
`} else if (name.includes("type_result")) {`

Reply:
Sorry for the mess, fixed it